### PR TITLE
Fix Audio Issues and Update to 1.0.0-a.30

### PR DESCRIPTION
**Pull request by eekrain**
Hey, i noticed in some webpages, when im playing video / streaming, it doesn't have the audio. The only page i test that has audio is just youtube. I added some runtimeLibs following nixpkgs's firefox/common.nix. For context iam using hyprland.

### CODE REVIEW

**Comment by yolofanhd** [comment]
a related issue to this pr: #10

File: flake.nix [hunk 1]
```diff
@@ -5,83 +5,129 @@
...
+        libxcb
+        libX11
+        libXcursor
+        libXrandr
+        libXi
+        libXext
+        libXcomposite
+        libXdamage
+        libXfixes
+        libXScrnSaver
```
Comment (MarceColl) [inline flake.nix]: Please, let's no introduce formatting changes in between actual changes, it makes checking what checked a lot harder than it needs to be :)

File: flake.nix [hunk 2]
```diff
@@ -5,83 +5,129 @@
...
+        libGL
+        libGLU
+        libevent
+        libffi
+        libjpeg
+        libpng
+        libstartup_notification
+        libvpx
+        libwebp
+        nasm
```
Comment (MarceColl) [inline flake.nix]: why do we need nasm?

**Review by MarceColl — CHANGES_REQUESTED** [review]

File: flake.nix (hunk 1 above)
Comment (eekrain) [reply flake.nix]: i am so sorry alejandra is tied to my save keybind

File: flake.nix (hunk 2 above)
Comment (eekrain) [reply flake.nix]: as i said i just follows nixpkgs's firefox/common. tbh i dont test it wheter it's working without nasm. wait let me remove it and test it

**Comment by eekrain** [comment]
I just reverted to the original formatting. And 1.0.0-a.30 was just dropped and i update it to it. nasm removed and turns out its fine. Let me know if there's more needs to fix

**Pull request closed without merging**